bash/t/terraform/terraform_run.functions

```diff
@@ -331,7 +331,7 @@ t_prefer_tf_env_var() {
 t_use_which_if_tf_var_not_set () {
     (
         TERRAFORM="" ;
-        which() {
+        command() {
```

Comment:
Why do you want to replace _which_ with _command_? Is that a Mac thing?

I don't understand the need for this specific change?

I'm overriding the installed `which` command in the `tf()` function so that I can effectively stub terraform for the sake of the automated tests.

Reply:
The only reason is `which`'s runtime is longer than `command` also `which` relies on PATH and couple of dependencies but `command` is shell's built-in

```
$ type command
command is a shell builtin

$ type which
which is /usr/local/bin/which
```
In the end `command` is more reliable than `which`

Reply:
Good reason. 👍